pywinauto/unittests/test_application.py

```diff
@@ -994,7 +994,7 @@ def tearDown(self):
         def test_folder_list(self):
             """Test that ListViewWrapper returns correct files list in explorer.exe"""
             files_list = self.desktop.MFC_samplesDialog.Shell_Folder_View.Items_View.wrapper_object()
-            self.assertEqual(files_list.texts(),
+            self.assertEqual([item.window_text() for item in files_list.get_items()],

```

Comment:
`files_list.texts()` does almost the same. Though it doesn't make difference because it's test code.